pretix_passbook/passbook.py

```diff
@@ -253,7 +253,10 @@ def generate(self, order_position: OrderPosition) -> Tuple[str, str, str]:
         passfile.description = gettext('Ticket for {event} ({product})').format(event=ev.name, product=ticket)
         passfile.barcode = Barcode(message=order_position.secret, format=BarcodeFormat.QR)
         passfile.barcode.altText = order_position.secret
-        passfile.relevantDate = ev.date_from.astimezone(tz).isoformat()
+        if order.event.settings.show_date_to and ev.date_to:
```

Comment:
Should we check, if date_from and date_to are on the same day? And if so, then keep using relevantDate? That way for one-day events with timestamps, the relevantDate is there for Apple Wallet to evaluate and show the ticket prominently.